app/views/layouts/_list_grid.html.haml

```diff
@@ -73,5 +73,8 @@
                                       :alt     => cell[:title]}
                 = cell[:text]
             - else
-              = image_tag(cell[:image], :alt => cell[:title], :title => cell[:title]) if cell[:image]
+              -# :icon supersedes :image
+              = image_tag(cell[:image], :alt => cell[:title], :title => cell[:title]) if cell[:image] && cell[:icon].nil?
+              - if cell[:icon].present?
```

Comment:
confused why you use icon.nil above and icon.present here
seems if this is precedence between these 2 images you'd have both in a conditional (e.g. `? :`, `if`, or  `||`)